build/Uno.WinUI.nuspec

```diff
@@ -122,6 +122,7 @@
 		<reference file="Uno.UI.dll" />
 		<reference file="Uno.UI.Composition.dll" />
 		<reference file="Uno.UI.Dispatching.dll" />
+		<reference file="Uno.UI.XamlHost" />
```

Comment:
The Uno.UI.XamlHost should be in a different package. We'll need to create a different nuspec for it.